File: iOSbeautyOfDevelop/chapter4/4.4/scrollView/scrollView/ViewController.m
//
//  ViewController.m
//  scrollView
//
//  Created by aa64mac on 8/16/15.
//  Copyright (c) 2015 cc. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIScrollView *mainScrollView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    NSArray *images = @[@"image0.png", @"image1.png", @"image2.png"];
    for (int i = 0; i < [images count]; i++) {
        UIImageView *imageV = [[UIImageView alloc]initWithFrame:CGRectMake(320 * i, 0, 320, 480)];
        
        imageV.contentMode = UIViewContentModeScaleAspectFit;
        
        NSString *filename = [NSString stringWithFormat:@"image%d.png", i];
        UIImage *image = [UIImage imageNamed:filename];
        
        imageV.image = image;
        
        [self.mainScrollView addSubview:imageV];
    }
    self.mainScrollView.contentSize = CGSizeMake(320 * 3, 80);
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: iOSDevGuide/chapter09/9.3.1/imgFrameAndbounds/imgFrameAndbounds/ViewController.swift
//
//  ViewController.swift
//  imgFrameAndbounds
//
//  Created by aa64mac on 10/21/15.
//  Copyright © 2015 cyj. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let image1 = UIImage(named: "img1.png")
        let imageView1 = UIImageView(image: image1)
        let imageView2 = UIImageView(image: image1)
        imageView1.frame = CGRectMake(10, 10, 320, 460)
        imageView2.frame = CGRectMake(60, 10, 160, 320)
        print("frame\(imageView1.frame)")
        print("bounds\(imageView1.bounds)")
        //imageView1.bounds = CGRectMake(100, 100, 160, 320)
        //imageView1.bounds = CGRectMake(0, 0, 160, 320)
        imageView1.bounds = CGRectMake(50, 0, 100, 460)
        print("frame\(imageView1.frame)")
        print("bounds\(imageView1.bounds)")
        
        imageView1.addSubview(imageView2)
        self.view.addSubview(imageView1)
        //imageView1.image =
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

